server/views/pages/contacts/create/checkAnswers.njk

```diff
@@ -8,25 +8,10 @@
 {% set mainClasses = "app-container govuk-body" %}
 
 {% block content %}
-<div class="govuk-grid-row govuk-!-static-margin-bottom-5">
-  <div class="govuk-grid-column-full">
-      {{ govukBreadcrumbs({
-      items: [
-          {
-            text: "Digital Prison Services",
-            href: DPS_HOME_PAGE_URL
-          },
-          {
-              text: "Contacts",
-              href: "/"
-          }
-      ]
-      }) }}
-  </div>
-</div>
```

Comment:
Has this been a decision now? Limit to 2 levels of breadcrumb? Thought we were still waiting on this, but fine either  way..

Reply:
No I made the change to add the "Prisoner Context" breadcrumb on all pages and de-duped it, it's now included in an import. After that is when I heard we might not even have breadcrumbs on these pages. Still, it's better than the broken ones we had before.